docs/Installation.md

```diff
@@ -16,9 +16,9 @@ _Linux Build Support_ component when installing Unity.
         width="500" border="10" />
 </p>
 
-## Clone the ml-agents Repository
+## Clone the Ml-Agents Toolkit Repository
```

Comment:
This change is actually incorrect. The repository is literally called `ml-agents`.